ingest-app/src/cmr/ingest/validation/generic_document_validation.clj

```diff
@@ -24,58 +24,52 @@
   []
   (mem-cache/create-in-memory-cache :ttl {} {:ttl SCHEMA_CACHE_TIME}))
 
-(defn- fetch-existing-concepts
-  "Fetch existing concepts of the given type from metadata-db for uniqueness validation"
-  [context concept-type]
-  (try
-    (mdb/find-concepts context {:latest true} concept-type)
-    (catch Exception e
-      (error (str "Error fetching concepts for generic validations: " (.getMessage e)))
-      []))) 
+(defn- make-params-from-uniqueness-fields
+  "Creates search parameters from the uniqueness fields defined in the schema.
+   Converts JQ-style field paths to lowercase, hyphenated keywords for search parameters."
+  [metadata fields]
+  (reduce (fn [params field-path]
+            (let [field-parts (generics/jq->list field-path keyword)
+                  field-value (get-in metadata field-parts)
+                  ;; Get the last part of the path and convert to parameter name format
+                  ;; TODO make search parameters part of the config.json for a concept instead of being hardcoded.
+                  param-name (-> field-parts
+                                 last
+                                 name
+                                 (string/replace #"([a-z])([A-Z])" "$1-$2")
```

Comment:
I need to fix this to use kebab case utility function

Reply:
also investigating if I still need last,name and lowercase, I think csk/->kebab-case will do all of that